Makefile

```diff
@@ -4,8 +4,18 @@ requirements: export CUSTOM_COMPILE_COMMAND=make requirements
 requirements:
 	pip-compile -o requirements/requirements.txt requirements/requirements.in
 	pip-compile -o requirements/prod-requirements.txt requirements/prod-requirements.in --allow-unsafe
+	pip-compile -o requirements/build-requirements.txt requirements/build-requirements.in
+	pip-compile -o requirements/dev-requirements.txt requirements/dev-requirements.in
+	pip-compile -o requirements/docs-requirements.txt requirements/docs-requirements.in
+	pip-compile -o requirements/test-requirements.txt requirements/test-requirements.in
```

Comment:
what's the difference between `build` and `test`? Perhaps a comment at the top of each file outlining its use would be helpful.

Reply:
I'm not sure, @dannyroberts ?

Reply:
@snopoke yeah I'm actually not sure either. `build-requirements.txt` isn't mentioned anywhere in either `commcare-hq` or `commcare-cloud` so it's possible that it's not used for anything really

Reply:
Hasn't been touched since 2013, and anything actively used would probably have seen some changes since then. I vote we kill it

Reply:
Done:
https://github.com/dimagi/commcare-hq/pull/21411/commits/9607caaa2a6da8e2b09c1dd16f28004d3754b306
https://github.com/dimagi/commcare-hq/pull/21411/commits/d18043564f0119a0a2731919b710f5ae86033d46